<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Century Gothic&quot;;
background-color:rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>763</width>
      <height>298</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Video</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Umami</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sour</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QProgressBar" name="salty_bar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sweet</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="bitter_bar">
           <property name="styleSheet">
            <string notr="true">QProgressBar {
     background-color: rgb(200, 200, 200);
 }</string>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Bitter</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QProgressBar" name="umami_bar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Salty</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QProgressBar" name="sweet_bar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="sour_bar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">
font: 75 14pt &quot;Century Gothic&quot;;
font-weight: bold;
color:rgb(200, 200, 200);</string>
           </property>
           <property name="text">
            <string>Taste Detected: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="6">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="6" column="1">
        <widget class="QProgressBar" name="disgust_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="happy_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="suprise_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QProgressBar" name="anger_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Happiness/Joy</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sadness</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QProgressBar" name="sad_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QProgressBar" name="fear_bar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Fear</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Surprise</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Anger</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;Century Gothic&quot;;
font-weight: bold;
color:rgb(200, 200, 200);</string>
         </property>
         <property name="text">
          <string>Emotion Detected: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Disgust</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
